node/coinstacks/solana/api/src/models.ts

```diff
@@ -1,14 +1,15 @@
+import { EnrichedTransaction } from 'helius-sdk'
 import { BaseAccount, BaseTx, BaseTxHistory } from '../../../common/api/src' // unable to import models from a module with tsoa
 
 /**
  * Contains info about a transaction
  */
-export interface Tx extends BaseTx {}
+export type Tx = EnrichedTransaction & BaseTx
 
 /**
  * Contains info about transaction history
  */
-export type TxHistory = BaseTxHistory<Tx>
+export type TxHistory = BaseTxHistory<Tx> & { cursor: string | undefined }
```

Comment:
`cursor` exists on `BaseTxHistory`
```suggestion
export type TxHistory = BaseTxHistory<Tx>
```

Reply:
Damn, nested types!

Fixed in [56a2d0f](https://github.com/shapeshift/unchained/pull/1052/commits/56a2d0f439b992653c76c8799b3e403a563dfd20)